### ISSUE TITLE
Handle all projects for transfer-cards utility

### ISSUE THREAD

**Issue by seemethere** [seemethere/release-bot#5]
Not all projects are returned due to github's pagination of results for the transfer-cards utility leading to false negatives when searching for specific projects.

Example:

```
$ GITHUB_TOKEN=***7d7f5 ./transfer-cards --repo-name=release-tracking 17.09.0-ce-rc2 17.09.0-ce-rc3
ERRO[0000] Source project '17.09.0-ce-rc2' not found in repo docker/release-tracking
```

We need a way to get all paginated results for projects.

**Comment by seemethere** [comment]
@corbin-coleman ^

**Comment by seemethere** [comment]
Closed with #6
